### ISSUE TITLE
Surface "artifacts" under stage details for a given pipeline

### ISSUE THREAD

**Issue by lwander** [spinnaker/spinnaker#2112]
This is for helping users see exactly which artifacts a pipeline was supplied with, and which artifacts a given stage has produced. Very helpful when an artifact supplied "provenance", giving a link back into whatever system created it.

**Comment by ezimanyi** [comment]
@sbwsg : This might be a duplicate of your work on surfacing artifact details

**Comment by lwander** [comment]
I think was actually the intended tracking bug :) There is still work to be done on apply those stage details to other artifact-enabled stages before we can close this, however.

**Comment by spinnakerbot** [comment]
This issue hasn't been updated in 91 days, so we are tagging it as 'stale'. If you want to remove this label, comment:

> @spinnakerbot remove-label stale

**Comment by spinnakerbot** [comment]
This issue is tagged as 'stale' and hasn't been updated  in 45 days, so we are tagging it as 'to-be-closed'. It will be closed in 45 days unless updates are made. If you want to remove this label, comment:

> @spinnakerbot remove-label to-be-closed

**Comment by spinnakerbot** [comment]
This issue is tagged as 'to-be-closed' and hasn't been updated  in 45 days, so we are closing it. You can always reopen this issue if needed.
